app/assets/stylesheets/application.scss

```diff
@@ -18,6 +18,8 @@ body {
   height: 100%;
 }
 
+.clearfix {clear:both;}
```

Comment:
without this, the form looks like this: (notice the cancel button is not cleared for some reason)
![screen shot 2015-06-18 at 12 19 51 pm](https://cloud.githubusercontent.com/assets/706359/8228027/5e53feb2-15b4-11e5-982c-6b52dbe75012.png)